src/Access/examples/kerberos_init.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
```

Comment:
I think that to make this example useful, we should describe how to see via klist what really happened.
Otherwise it is not needed.

Reply:
Described kerberos_init example in comments and mentioned klist command.